packages/eslint-plugin-office-addins/README.md

```diff
@@ -0,0 +1,44 @@
+# eslint-plugin-office-addins
+
+eslint plugin for office-addins
+
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
```

Comment:
This seems wrong. Shouldn't users install office-addin-lint rather than use eslint directly?